spec/aws_support/matchers/create_an_aws_object.rb

```diff
@@ -41,7 +41,7 @@ def match_failure_messages(recipe)
         differences += match_values_failure_messages(example.be_updated, recipe, "recipe")
 
         # Check for object existence and properties
-        resource = resource_class.new(name, nil)
+        resource = resource_class.new(name, recipe.client.run_context)
```

Comment:
How necessary is this? I worry a tiny bit at giving the resource the ability to muck with the existing run_context. Not hugely, but some.